pyproject.toml

```diff
@@ -94,3 +96,12 @@ markers = [
 
 [tool.flit.module]
 name = "astro"  # Or "astro.sql" if you just want this directory, not the entire 'astro'.
+
+[tool.mypy]
+files = [
+    "src/astro/settings.py",
+    "src/astro/sql/__init__.py",
+    "src/astro/sql/operators/agnostic_aggregate_check.py",
+    "src/astro/sql/operators/agnostic_boolean_check.py"
```

Comment:
Once we complete  #281 we can replace the `files` parameter by the `mypy_path` one:
https://mypy.readthedocs.io/en/stable/config_file.html#import-discovery

Reply:
Should we add more checks similar to the one I added in https://github.com/astro-projects/astro/pull/295/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R104 ?

```
namespace_packages = true
explicit_package_bases = true
pretty = true
show_error_codes = true
ignore_missing_imports = true
no_implicit_optional = true
warn_redundant_casts = true
show_error_context = true
color_output = true
disallow_any_generics = true
disallow_untyped_defs = true
disallow_incomplete_defs = true
warn_unused_configs = true
warn_unused_ignores = true
warn_return_any = true
strict_equality = true
```